config/CMakeAddFortranSubdirectory.cmake

```diff
@@ -250,16 +250,9 @@ function(cmake_add_fortran_subdirectory subdir)
     CONFIGURE_COMMAND ${CMAKE_COMMAND} -DCMAKE_BUILD_TYPE=${ep_build_type}
                       -P ${build_dir}/config_cafs_proj.cmake
     BUILD_COMMAND     ${CMAKE_COMMAND} -P ${build_dir}/build_cafs_proj.cmake
+    BUILD_ALWAYS 1
```

Comment:
Is this related to CUDA changes or something for debugging?

Reply:
This is related to an upstream fix in CMake that was a result of a bug I reported.  `CMakeAddFortranSubdirectory.cmake` is a highly modified version of a standard CMake capability. (Yes. I should submit some of my additions back to CMake.)

Reply:
Oh - in the MSVC builds, I need to use a pre-release of CMake to avoid a bug in CMake.  This change is required to allow the code to build correctly when using this pre-release.  It does not impact the Linux build.